### PR TITLE
Update tutorial one to reference gatsby astronaut image

### LINKED ISSUE #9655: Update tutorial one to use the cute gatsby illustration
#9412 updates the default starter to use a cool illustration of an astronaut holding a tasty treat! Let's update the tutorial one to use the same image, instead of a random image from unsplash. 
The tutorial is here:
https://www.gatsbyjs.org/tutorial/part-one/
The file to edit is here:
https://github.com/gatsbyjs/gatsby/blob/master/docs/tutorial/part-one/index.md

You will also need to update the relevant screenshots.

Let's spread the purple love!

### PR DESCRIPTION
**Pull request by ebello**
This will close #9655 but should not be merged until `gatsby-starter-hello-world` adds the referenced image, which @barrymcgee did in https://github.com/gatsbyjs/gatsby-starter-hello-world/pull/27.

### CODE REVIEW

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -79,16 +79,17 @@ export default () => (
 
 ![More changes with hot reloading](03-more-hot-reloading.png)
 
-4.  Add an image. (In this case, a random image from Unsplash).
+4.  Add an image. (In this case, the Gatsby astronaut).
 
-```jsx{7}:title=src/pages/index.js
+```jsx{2,8}:title=src/pages/index.js
```
Comment (barrymcgee) [inline docs/tutorial/part-one/index.md]: Can you explain to me what this change is please? I haven't seen it before...

Comment (ebello) [reply docs/tutorial/part-one/index.md]: This is code snippet line highlighting. It specifies that line 2 and 8 should be highlighted to denote the changes made.

**Comment by ebello** [comment]
Not a problem, this is even easier which is the spirit of the tutorial. Just pushed.

File: docs/tutorial/part-one/index.md (hunk 1 above)
Comment (barrymcgee) [reply docs/tutorial/part-one/index.md]: TIL - thank you.

**Comment by barrymcgee** [comment]
@ebello Looks good to me - once my PR to add the image to the starter lands, this can land too 👌🏻

**Comment by ebello** [comment]
Hi @shannonbux, we have an outstanding PR from @barrymcgee to add the image to the starter here: 
https://github.com/gatsbyjs/gatsby-starter-hello-world/pull/27.

I like your idea of using the image with `gatsby-image`, though I think that might be a more advanced step. Introducing the `static` folder here as a place to put assets like the image might be a more useful building block at this stage. Perhaps we could add a bit of text in the tutorial that introduces the notion of optimizing images and link people off to the "[Using gatsby-image](https://www.gatsbyjs.org/docs/using-gatsby-image/)" guide?

**Comment by shannonbux** [comment]
Hello everyone! Thanks for your hard work here and apologies for the time it took to reach a decision on this PR and the connected one. For now, I'm going to close this in favor of https://github.com/gatsbyjs/gatsby/issues/9374 since a adding a page to the tutorial specifically on `gatsby-image` introduces a best practice. Thanks in advance for your patience with this idea; we only recently realized how few people know about `gatsby-image` and we'd like to use the tutorial as one of the ways to teach about it.

**Comment by barrymcgee** [comment]
Hi @shannonbux - thanks for bringing this to a conclusion. Just so I know for future, what's the process around approving issues so we know that when we start an issue, it's a valid/approved issue?

I, and I'm sure @ebello, would like to avoid this situation in future where we spent time working on an issue that it seems no-one agreed it should have been worked on at all.

If I get some time this week, I'll have a look at the `gatsy-image` tasks.

**Comment by shannonbux** [comment]
Hi!

Thanks for responding and asking questions! In this case, it was totally
legitimate for you to propose the issue and then it was the core team’s
responsibility to evaluate the issue and it took us some time to
deliberate; we are also committed on reducing that amount of time in the
future. This is part of the beauty of open source projects—there are
creative ideas from the community that inspire valuable discussions! So
imho this issue has been super valuable for Gatsby community because it
spurred discussions on:
- what the purpose of the tutorial is
- how to encourage best practices in general
- how to make it easier for people to learn and use gatsby-image
- how to streamline decision-making about edits to the tutorial in the
future

In other words, I’m really glad you opened the issue and worked on the PR
because it made a big difference.

To address your question on how to avoid creating this same situation in
the future (and thanks for asking and holding us accountable! It’s awesome
to know that you want to make sure we optimize our open source process over
time), here is the text I drafted yesterday to save as a GitHub saved reply
and also I just submitted a PR to put this in the “How to Contribute”
documentation. This will hopefully help clarify the process. Would be
valuable to get feedback on this too!

“Hello! The Gatsby open source community is committed to increasing the
quality of Gatsby documentation, and this contribution needs to match our
contribution criteria. We kindly ask that you make sure this contribution
meets the standards outlined in the [Gatsby Style Guide](
https://www.gatsbyjs.org/docs/gatsby-style-guide/). Also, if this PR did
not come from an issue written by the core team, please add a comment to
this PR that explains why this should be included in the docs, according to
the [Docs Decision Tree](
https://www.gatsbyjs.org/blog/2018-10-12-uptick-docs-contributions-hacktoberfest/#docs-decision-tree-and-examples
).

If, after two weeks the two requirements above haven't been met, this issue
and/or PR will be closed, which helps us triage issues and PRs efficiently.
You can request that it be reopened when and if you are ready to make the
updates required.

Thank you in advance and cheers for contributing to Gatsby! It's people
like you that make this community great! :smiling_face:”

Thanks again for responding and asking questions!

**Pull request closed without merging**